doc/source/reference/arrays.nditer.rst

```diff
@@ -209,6 +201,47 @@ raise an exception
       File "<stdin>", line 1, in <module>
     ValueError: Iterator flag EXTERNAL_LOOP cannot be used if an index or multi-index is being tracked
 
+Alternative Looping and Element Access
+--------------------------------------
+
+To make its properties more readily accessible during iteration,
+:class:`nditer` has an alternative syntax for iterating, which works
+explicitly with the iterator object itself. With this looping
+construct, the current value is accessible by indexing into the
+iterator. Other properties, such as tracked indices remain as before.
+
+The Python interactive interpreter unfortunately prints out the
```

Comment:
I don't understand this paragraph. I know you just copied it from above, but can we just remove it altogether?

Reply:
I had no idea what it was, so I left it just in case. Glad it's not just me and will remove.